db-ops/create-db-user.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-# docker image with postgres.mysql client
-DOCKER_IMAGE_TAG=commitdev/zero-k8s-utilities:0.0.3
+# docker image with postgres.mysql client only
+DOCKER_IMAGE_TAG=commitdev/zero-k8s-utilities:mysql-postgres
```

Comment:
should probably use `0.0.3`since its a newer tag, we want to phase out the `named-tags`